server/embed/config.go

```diff
@@ -1084,8 +1102,14 @@ func (cfg *Config) Validate() error {
 		return fmt.Errorf("setting experimental-enable-lease-checkpoint-persist requires experimental-enable-lease-checkpoint")
 	}
 
-	if cfg.ExperimentalCompactHashCheckTime <= 0 {
-		return fmt.Errorf("--experimental-compact-hash-check-time must be >0 (set to %v)", cfg.ExperimentalCompactHashCheckTime)
+	if cfg.ExperimentalCompactHashCheckTime < 0 {
+		return fmt.Errorf("--experimental-compact-hash-check-time must be >=0 (set to %v)", cfg.ExperimentalCompactHashCheckTime)
+	}
```

Comment:
Is this a bug? Let's fix it separately, and it should also be backported to 3.5.